shared/dataflow/codeql/dataflow/internal/DataFlowImpl.qll

```diff
@@ -2487,6 +2486,264 @@ module MakeImpl<LocationSig Location, InputSig<Location> Lang> {
           callEdgeReturn(call, c, _, _, _, _, _)
         }
 
+        additional module Graph {
```

Comment:
Perhaps add QL doc that this is for debugging only.

Reply:
It's only exposed through the `Debug` module. But a guess an additional piece of qldoc here improves local readability. I'll add it.